runtime/wasm/src/module/test/abi.rs

```diff
@@ -365,3 +365,23 @@ fn abi_big_int() {
     let new_uint_from_u256 = BigInt::from_signed_u256(&new_uint.to_signed_u256());
     assert_eq!(new_uint, new_uint_from_u256);
 }
+
+// This should panic rather than exhibiting UB. It's hard to test for UB, but
+// when reproducing a SIGILL was observed which would be caught by this.
```

Comment:
Pardon my ignorance: what's a SIGILL?

Reply:
It's kinda like a segfault. It's a signal that's fired when the CPU tries to execute an invalid instruction, for example if you try to execute data.

Reply:
Wow, in all my days of writing C and other low-level languages, I've never come across it. Thanks for clarifying!